tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_scanner.c

```diff
@@ -67,7 +67,7 @@ static void test_tx_device_setup(void)
 /** Bypassing setting up transmission, and will try to send the raw data that is
  *  provided to the function.
  */
-static void test_tx_send_ad_type_msg(uint8_t type, uint8_t *data, uint8_t len)
+static void test_tx_send_ad_type_msg(uint8_t type, const uint8_t *data, uint8_t len)
```

Comment:
Could you split this into a separate commit (in this same PR however)?

Reply:
It has been done in this PR https://github.com/pulls
added DNM label to wait related PR merged first. Add this change just to pass PR checks.

Reply:
Got it. Thanks for the clarification.